### PR TITLE
Change job scenario and action for OSP13 gating

### PR DESCRIPTION
**Pull request by hmakkapati**

### CODE REVIEW

**Comment by hmakkapati** [comment]
These changes are to enable https://github.com/rcbops/rpc-gating/pull/1224
Once this and the rpc-gating PR land, we can get rid of the legacy stuff

File: tests/test-ansible-functional.sh [hunk 1]
```diff
@@ -81,7 +81,7 @@ esac
 # ansible that is not available in liberty and mitaka.  There is no reason
 # to run it in a ceph context either.
 case $RE_JOB_SCENARIO in
-  kilo|liberty|mitaka|ceph|hummingbird|3ctlr_2comp_3ceph)
+  kilo|liberty|mitaka|ceph|hummingbird|3ctlr_2comp_3ceph|osp13)
```
[the remark this answers is not in the document]
Comment (npawelek) [reply tests/test-ansible-functional.sh]: Actually, this likely needs to be changed after the rpc-gating/pull/1224 PR merges

**Comment by npawelek** [comment]
There looks to be a failure registering RHN. The `{{ redhat_osp_version }}` variable appears to be set incorrectly? 

The var:
```playbooks/deploy-osp.yml:        - rhel-7-server-openstack-{{ redhat_osp_version }}-rpms```

The error:
```failed: [director] (item=rhel-7-server-openstack-3ctlr_2cm_3ceh-rpms) => {"changed": false, "item": "rhel-7-server-openstack-3ctlr_2cm_3ceh-rpms", "msg": "rhel-7-server-openstack-3ctlr_2cm_3ceh-rpms is not a valid repository ID", "results": ["rhel-7-server-openstack-3ctlr_2cm_3ceh-rpms is not a valid repository ID"]}```

File: tests/test-ansible-functional.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (hmakkapati) [reply tests/test-ansible-functional.sh]: Right, that's the plan.

**Comment by hmakkapati** [comment]
recheck_all

**Comment by hmakkapati** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

**Comment by hmakkapati** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

**Comment by hmakkapati** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

File: tests/test-setup.yml [hunk 2]
```diff
@@ -48,6 +48,12 @@
         dest: "/etc/openstack_deploy/user_rpcm_{{ ansible_env.RE_JOB_ACTION | default('master') }}_vars.yml"
       when: (deploy_osp | default(False))
```
[the remark this answers is not in the document]
Comment (hmakkapati) [reply tests/test-setup.yml]: The source file should exist. 
You mean the destination file won't be consumed, @tonytan4ever ?

**Comment by hmakkapati** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

File: tests/test-setup.yml (hunk 2 above)
[the remark this answers is not in the document]
Comment (tonytan4ever) [reply tests/test-setup.yml]: Oh yes, you're right.  
My confusion comes from the fact we have to copy the variable overrides twice for osp-13 gates, but I guess that's just a temporary measure.  
I will merge after it gates.

**Comment by tonytan4ever** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

**Comment by tonytan4ever** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

**Comment by tonytan4ever** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

**Comment by tonytan4ever** [comment]
recheck_bionic_mnaio_3ctlr_2comp_3ceph_osp_13_deploy

**Pull request merged**